orc8r/cloud/docker/fluentd_daemon/Gemfile

```diff
@@ -9,9 +9,9 @@ gem "fluent-plugin-prometheus", "~> 1.5.0"
 gem 'fluent-plugin-json-in-json-2', ">= 1.0.2"
 gem "fluent-plugin-record-modifier", "~> 2.0.0"
 gem "fluent-plugin-rewrite-tag-filter", "~> 2.2.0"
-gem "elasticsearch", "~> 7.1"
-gem "fluent-plugin-elasticsearch", "~> 3.5.5"
+gem "elasticsearch", "~> 7.13.0"
+gem "fluent-plugin-elasticsearch", "~> 5.2.1"
```

Comment:
This probably just takes the versions that are installed in the Dockerfile already?

Reply:
Yes. For some reason, the versions for fleuntd_daemon were older than for fluentd_forward, not sure if that was intended or not, but I set everything to the same version now. But the Dockerfiles had to be changed at some parts as well. The main point is that the older versions of elasticsearch and the fluentd-plugin do not support the newest version of fluentd.